.buildkite/pipelines/on_merge.yml

```diff
@@ -37,7 +37,7 @@ steps:
       image: family/kibana-ubuntu-2004
       imageProject: elastic-images-prod
       provider: gcp
-      machineType: n2-standard-2
+      machineType: n2-highcpu-8
```

Comment:
during a run I managed to observe, we used ~60% max of the CPU through the run. <50% of the memory.
We can possibly go even lower to `n2-highcpu-4`, to save a few bucks, and only add 1-3 min extra?

Reply:
Yeah we've had issues with that elsewhere - we can run jest on n2-2-spot but the preemption rate is so high it becomes unreliable.  Unfortunate.